backend/store/etcd/check_store_test.go

```diff
@@ -39,121 +39,18 @@ func TestCheckConfigStorage(t *testing.T) {
 		assert.Equal(t, check.Command, retrieved.Command)
 		assert.Equal(t, check.Stdin, retrieved.Stdin)
 
-		checks, continueToken, err = store.GetCheckConfigs(ctx, 0, "")
+		checks, err = s.GetCheckConfigs(ctx, pred)
 		assert.NoError(t, err)
 		assert.NotEmpty(t, checks)
 		assert.Equal(t, 1, len(checks))
-		assert.Empty(t, continueToken)
+		assert.Empty(t, pred.Continue)
+		for _, check := range checks {
+			fmt.Printf("TestCheckConfigStorage: %#v\n", check)
+		}
```

Comment:
I'm assuming that's a debug `for` loop that can be removed?

Reply:
I'm impressed your debug print statements contain no vulgarity 😆

Reply:
😬